lightningd/opening_control.c

```diff
@@ -212,7 +212,9 @@ wallet_commit_channel(struct lightningd *ld,
 			      true,
 			      &uc->local_basepoints,
 			      &uc->local_funding_pubkey,
-			      NULL);
+			      NULL,
+			      feerate,
```

Comment:
This is most likely the wrong feerate. The `feerate` variable concerns on-chain fees for the funding transaction, not the base fee leveraged off-chain.

Reply:
not in final version, redoing branch